externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -70,16 +71,19 @@ case class SparkSQLEngine(spark: SparkSession) extends Serverable("SparkSQLEngin
 
 object SparkSQLEngine extends Logging {
 
+  val sparkConf: SparkConf = new SparkConf()
+
   val kyuubiConf: KyuubiConf = KyuubiConf()
 
   var currentEngine: Option[SparkSQLEngine] = None
 
   private val user = currentUser
 
+  val appName = s"kyuubi_${user}_spark_${Instant.now}"
```

Comment:
unnecessary change

Reply:
> unnecessary change

`appName` is used in both `setupConf` method and `createSpark` method.

Reply:
that‘s true. but the appName is not right here. for a complete pipeline, the appName is generated by the Kyuubi server or set by users.

I suggest that we simply name the groupId in `createSpark` as "engine initializing queries "

Reply:
> that‘s true. but the appName is not right here. for a complete pipeline, the appName is generated by the Kyuubi server or set by users.
> 
> I suggest that we simply name the groupId in `createSpark` as "engine initializing queries "

Got it, i will do that.